### ISSUE TITLE
Add support for aria role="radiogroup"

### ISSUE THREAD

**Issue by mfulton26** [twbs/bootstrap#25230]
It would be great if buttons.js managed `aria-checked` whenever an element is marked with `role="radio"` as a child to an element marked as `role="radiogroup"`.

```html
<div class="btn-group" data-toggle="buttons" role="radiogroup">
  <label class="btn btn-primary active" role="radio" aria-checked="true">
    <input type="radio" name="options" id="option1" autocomplete="off" checked> Radio 1 (preselected)
  </label>
  <label class="btn btn-primary" role="radio" aria-checked="false">
    <input type="radio" name="options" id="option2" autocomplete="off"> Radio 2
  </label>
  <label class="btn btn-primary" role="radio" aria-checked="false">
    <input type="radio" name="options" id="option3" autocomplete="off"> Radio 3
  </label>
</div>
```

This would allow screen readers to better communicate radio button groups to users and allow a radiogroup widget to be labeled, etc. improving the accessibility of any containing form, etc.
  

See also [ARIA Radio Group and Radio Example](https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/radio/radio.html).

**Comment by patrickhlauke** [comment]
closing this, as unless i'm missing something, this example makes little sense...why would you need `aria-checked` on the labels if there's actual radio buttons? and nothing stops you from adding `role="radiogroup"` right now if you feel that's a better fit to identify the grouping
